processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -149,6 +150,17 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
+  public double estimateSelectivity(ColumnSelector columnSelector, BitmapIndexSelector indexSelector)
+  {
+    // Estimate selectivity with attribute value independence assumption
```

Comment:
Yes, this assumption might be wrong, maybe in many real applications. However, in that cases, users can simply use other search strategies like ```UseIndexStrategy``` or ```CursorOnlyStrategy```. This is why we still use ```UseIndexStrategy``` as the default strategy.